Calling/src/app/App.tsx

```diff
@@ -82,26 +83,22 @@ const App = (): JSX.Element => {
       document.title = `home - ${WEB_APP_TITLE}`;
       // Show a simplified join home screen if joining an existing call
       const joiningExistingCall: boolean = !!getGroupIdFromUrl() || !!getTeamsLinkFromUrl();
+
       return (
         <HomeScreen
           joiningExistingCall={joiningExistingCall}
-          startCallHandler={(callDetails) => {
+          startCallHandler={async (callDetails) => {
```

Comment:
is there a need for this to be marked async?

Reply:
(if its just to reduce the diff between sample and main then leave it in)

Reply:
This comes from upstream. I think it's because in some scenarios we now need the handler to be async?

In any case, I'll take a better look when I sync back the conditional compilation slips that I had to fix in the "manual fixes" commit here.